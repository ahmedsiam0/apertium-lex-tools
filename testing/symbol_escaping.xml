<rules>
	<rule>
		<match tags="num"><select tags="num.ord"/></match>
	</rule>
	<rule weight="3">
		<match lemma="$"/>
		<match tags="num">
			<select tags="num.card"/>
		</match>
	</rule>
</rules>
