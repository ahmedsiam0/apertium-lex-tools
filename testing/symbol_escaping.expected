^3<num>/3<num><ord>$
^\$<sym>/\$<sym>$ ^3<num>/3<num><card>$
